src/main/java/com/imsweb/layout/LayoutFactory.java

```diff
@@ -41,7 +42,13 @@
  */
 public final class LayoutFactory {
 
-    // constants for the internal layout IDs
+    //XML Layouts - constants for the internal layout IDs
+    public static final String LAYOUT_ID_NAACCR_XML_16 = "naaccr-xml-16"; // kept for backward compatibility, actually means abstract...
```

Comment:
Can you please replace all those "kept for backward compatibility..." comments by 
```java
// alias for full abstract
```
I think it's more accurate.